module.json

```diff
@@ -3,9 +3,9 @@
     "name": "Calendar",
     "description": "Calendar for spaces or user profiles.",
     "keywords": ["calendar"],
-    "version": "1.0.12",
+    "version": "1.0.13",
     "humhub": {
-        "minVersion": "1.4"
+        "minVersion": "1.7"
```

Comment:
I think you do not need to change the min version for this PR, especially 1.7 is not even released yet. You probably tested only with dev branch but I don't see any breaking changes in this PR.

Reply:
I got it